rollbar-spring-webmvc/build.gradle

```diff
@@ -0,0 +1,18 @@
+ext {
+    springWebmvcVersion = '5.2.6.RELEASE'
+    javaxServletVersion = '3.0.1'
+    junitVersion = '4.12'
+    mockitoVersion = '2.10.0'
+    springFrameworkBootTestVersion = '2.2.6.RELEASE'
+}
+
+dependencies {
+    compile project(':rollbar-web')
```

Comment:
I think that if we are using the new naming to scope dependencies we should replace this by `implementation` and `testCompile` by `testImplementation`.

Reply:
i think you mean change testCompile to testImplementation and compile to implementation

Reply:
for some reason just for :rollbar-web I need to use compile project. implementation doesn't pull in the rollbar-java dependencies